src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -13,8 +13,15 @@ <h2 class="card-pf-title">
       <span class="card-pf-aggregate-status-title">{{$ctrl.status.title}}</span>
     </span>
   </h2>
-  <div class="card-pf-body">
-    <p class="card-pf-aggregate-status-notifications">
+  <div class="card-pf-body" ng-class="{'show-spinner': $ctrl.showSpinner}" ng-style="$ctrl.spinnerHeight">
```

Comment:
the spinnerHeight ought only be used when the spinner is shown.

Reply:
Agree